<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CW3SelectViewLayout</class>
 <widget class="QWidget" name="CW3SelectViewLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Layout</string>
  </property>
  <widget class="QPushButton" name="m_btnMPRLayout">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="m_lblCol">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>183</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Col</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnSegLayout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>96</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnSingletonLayout">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>53</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spbSelectionRow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>203</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spbSelectionCol">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>203</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_lblRow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>183</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Row</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnSelectionLayout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>189</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnLeftmostLayout">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>139</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MPR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>106</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Segmentation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>63</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Single</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>149</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
